realm/realm-library/src/main/java/io/realm/internal/Table.java

```diff
@@ -354,7 +354,7 @@ public static void throwDuplicatePrimaryKeyException(Object value) {
     // Getters
     //
 
-    OsSharedRealm getSharedRealm() {
+    public OsSharedRealm getSharedRealm() {
```

Comment:
public because of `OsObjectBuilder`?

Reply:
Yes.

Last year when discussing internal architecture we decided that to have a cleaner separation of concerns and make it easier for new people to come to the code base we should move relevant code from internal into `internal.objectstore` and `internal.core`. OsObjectBuilder lives in `internal.objectstore` which means that some of the methods in internal are forced to become public.